pkg/apis/extensions/v1alpha1/rpaasinstance_types.go

```diff
@@ -21,6 +16,11 @@ type RpaasInstanceSpec struct {
 	// PlanName is the name of the rpaasplan instance.
 	PlanName string `json:"planName"`
 
+	// Host is the application address where all incoming HTTP will be
+	// forwarded for.
+	// +optional
+	Host string `json:"host,omitempty"`
```

Comment:
This would work for now, but since we're here what do you think about trying to future-proof this? I'm thinking of instead of a single `Host` having something like:
```go
Binds []Bind
```
```go
type Bind struct {
  Name string // Set to app name
  Host string
}
```
It would be okay for the manager to return an error if there's more than one bind as we do today. But multiple binds could be used in the future and exposed on blocks/locations templates for the user to deal with.

Reply:
I agree, this makes total sense. I'll improve this later.